string-manipulation-1/src/get-first-char.js

```diff
@@ -0,0 +1,4 @@
+/* exported getFirstChar */
+function getFirstChar(string) {
+  return string.charAt(0);
```

Comment:
Remember you can directly access a character in a string using bracket notation.

```JAVASCRIPT
return string[0];
```